staging/src/k8s.io/api/storage/v1beta1/types.go

```diff
@@ -321,6 +321,23 @@ type CSIDriverSpec struct {
 	// +optional
 	PodInfoOnMount *bool `json:"podInfoOnMount,omitempty" protobuf:"bytes,2,opt,name=podInfoOnMount"`
 
+	// Defines if the underlying volume supports changing ownership and
+	// permission of the volume before being mounted.
+	// If set to Always, SupportsFSGroup indicates that
+	// the volumes provisioned by this CSIDriver support volume ownership and
+	// permission changes, and the filesystem will be modified to match the
+	// defined fsGroup every time the volume is mounted.
+	// If set to Never, then the volume will be mounted without modifying
+	// the volume's ownership or permissions.
+	// Defaults to Evaluate, which results in the volume being examined
+	// and the volume ownership and permissions attempting to be updated
+	// only when the PodSecurityPolicy's fsGroup is explicitly defined, the
+	// fsType is defined, and the PersistentVolumes's accessModes includes RWO.
+	// This field is alpha-level, and is only honored by servers
+	// that enable the CSIVolumeSupportsFSGroup feature gate.
+	// +optional
+	SupportsFSGroup *SupportsFSGroup `json:"supportsFSGroup,omitempty" protobuf:"bytes,4,opt,name=supportsFSGroup"`
```

Comment:
Can you put this at the end so that it's in numbered order?

Reply:
I've moved this to the end in the `v1/types.go` and `v1beta1/types.go` files. I've adjusted it to be in alphabetical order (using the new name `FSGroupPolicy` in the `pkg/apis/storage/types.go` file. Let me know if this is incorrect.

Reply:
Sorry by numbered order I mean by the json index number